core/web/jobs_controller.go

```diff
@@ -137,20 +137,37 @@ func (jc *JobsController) Create(c *gin.Context) {
 // Example:
 // "DELETE <application>/specs/:ID"
 func (jc *JobsController) Delete(c *gin.Context) {
-	jobSpec := job.Job{}
-	err := jobSpec.SetID(c.Param("ID"))
+	j := job.Job{}
+	err := j.SetID(c.Param("ID"))
 	if err != nil {
 		jsonAPIError(c, http.StatusUnprocessableEntity, err)
 		return
 	}
 
-	err = jc.App.DeleteJob(c.Request.Context(), jobSpec.ID)
+	// Do not allow the job to be deleted if it is managed by the Feeds Manager
+	isManaged, err := jc.App.GetFeedsService().IsJobManaged(c.Request.Context(), int64(j.ID))
+	if err != nil {
+		jsonAPIError(c, http.StatusInternalServerError, err)
+
+		return
+	}
+
+	if isManaged {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("job must be deleted in the feeds manager"))
+
+		return
+	}
+
+	// Delete the job
+	err = jc.App.DeleteJob(c.Request.Context(), j.ID)
```

Comment:
Why not simply add the check for if its managed inside DeleteJob?

Reply:
Good point! Fixed